libs/blocks/caas/utils.js

```diff
@@ -8,6 +8,92 @@ const pageConfig = pageConfigHelper();
 const pageLocales = Object.keys(pageConfig.locales || {});
 const requestHeaders = [];
 
+const LOCALES = {
```

Comment:
Can we combine this and the LOCALES object in send-utils?  I don't think we should have two copies within the caas files.

Reply:
I know this was originally copied from the milo locales object, but I don't think we need the `ietf` subobject:
```
const LOCALES = {
  ar: 'es-AR',
  br: 'pt-BR',
  ...
};
```

Reply:
nit: Please alpha sort within each region